options/posix/include/bits/posix/timer_t.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _MLIBC_TIMER_T_H
+#define _MLIBC_TIMER_T_H
+
+typedef void * timer_t;
```

Comment:
Why is this a `void *`? What will it point to if POSIX timers are implemented?

Reply:
On both glibc and musl, `timer_t` is `typedef`ed as a `void *`.

Reply:
What does it point to?

Reply:
On musl, it points to a `intptr_t`. I remember reading that on glibc, it also points to some integer type.